package.json

```diff
@@ -43,12 +43,13 @@
     "react-tap-event-plugin": "^1.0.0"
   },
   "dependencies": {
+    "babel-runtime": "^6.11.6",
```

Comment:
@oliviertassinari Is this needed? it's already in the dependencies for the plugin: https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-runtime/package.json